spyder/plugins/remoteclient/api/modules/file_services.py

```diff
@@ -400,7 +401,7 @@ async def is_dir(self, path: Path):
             return await response.json()
 
     async def mkdir(
-        self, path: Path, create_parents: bool = True, exist_ok: bool = False
+        self, path: Path, *, create_parents: bool = True, exist_ok: bool = False
```

Comment:
```suggestion
        self,
        path: Path,
        *,
        create_parents: bool = True,
        exist_ok: bool = False
```

This line is longer than 79 columns.